content/en/docs/started/support.md

```diff
@@ -4,224 +4,99 @@ description = "Where to get support for Kubeflow"
-      </tr>
-      <tr>
-        <td><a href="https://www.seldon.io/">Seldon</a></td>
-        <td> 
-        <a href="https://github.com/SeldonIO/seldon-core/issues">Issue 
-        tracker</a></td>
-      </tr>    
-    </tbody>
-  </table>
-</div>
+## Support from commercial providers in the Kubeflow Ecosystem
 
-<a id="cloud-support"></a>
-## Support from a cloud or platform provider
+We want to promote commercial companies and idividuals that contribute back to the open source project.
+Below is a table of organizations that contribute to Kubeflow and offer commercial support:
 
-If you're using the services of a cloud provider to host Kubeflow, the cloud
-provider may be able to help you diagnose and solve a problem.
+| Provider                      | Support Link                                                                                                                                                    |
+|-------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aranui Solutions              | [Aranui Solutions](https://www.aranui.solutions/services)                                                                                                       |
+| Canonical                     | [Ubuntu Kubeflow](https://ubuntu.com/kubeflow#get-in-touch)                                                                                                     |
+| Freelancer Julius von Kohout  | [LinkedIn](https://de.linkedin.com/in/juliusvonkohout/), [Slack](https://cloud-native.slack.com/team/U06LW431SJF), [GitHub](https://github.com/juliusvonkohout) |
+| Red Hat                       | [Red Hat](https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-ai)                                                                        |
+| Other Providers               | Please reach out to the Kubeflow Steering Committee with proof of significant contributions to the Kubeflow open source project                                 |
 
-Consult the support page for the cloud service or platform that you're using:
-
-* [Amazon Web Services (AWS)](https://aws.amazon.com/contact-us/)
-* [Canonical Ubuntu](https://ubuntu.com/kubeflow#get-in-touch)
-* [Google Cloud Platform (GCP)](https://cloud.google.com/support-hub/)
-* [IBM Cloud](https://www.ibm.com/cloud/support)
-* [Microsoft Azure](https://azure.microsoft.com/en-au/support/options/)
-* [Red Hat OpenShift](https://help.openshift.com/)
-
-## Other places to ask questions
-
-You can also try searching for answers or asking a question on Stack Overflow. 
-See the [questions tagged with
-“kubeflow”](https://stackoverflow.com/questions/tagged/kubeflow).
+<a id="cloud-support"></a>
+If you are using a managed offer from a cloud provider for Kubeflow, then the cloud
```

Comment:
Would it make sense to link Kubeflow Distributions here ?
https://www.kubeflow.org/docs/started/installing-kubeflow/#packaged-distributions

Reply:
Not all of them offer commercial support, so i would leave this separate.